### PR TITLE
Created new staff page for 2020

### PR DESCRIPTION
**Pull request by websupergirl**
I have everyone up there that I had information for at this point - future people should be able to add easily due to switch in design.

For https://github.com/csjoblom/hackforacause-website/issues/98

### CODE REVIEW

File: pages/meet-our-team/index.vue [hunk 1]
```diff
@@ -34,6 +35,91 @@ export default {
...
+            {
+                name: "Nikole Gipps",
+                title: "WordPress Engineer",
+                company: "X-Team",
+                image: require('~/assets/images/nikole.jpg')
+            },
+            {
+                name: "Frankie Gold",
+                title: "Data Engineer",
+                company: "IDX Broker",
```
Comment (csjoblom) [inline pages/meet-our-team/index.vue]: Frankie is now at VM Ware as a Go Engineer

File: pages/meet-our-team/index.vue [hunk 2]
```diff
@@ -9,17 +9,18 @@
         <h3 class="h4c--title">Meet Our Team</h3>
       </div>
       <p class="lead text-center text-white">
-        Special thanks to Aaron Martins, Antonio Ortega Jr, Ashley
-        Espinoza, Chris Sjoblom, Diego Kourchenko, Eric Hadley, Eric
-        Wills, Jeff Petry, Lauren Neel, Lee Ralls, Mark Davis, Matt
-        Sayre, Shelly Galvin, Stephen Parac &amp; Sylvia Barry
+        The Hack for a Cause Steering Committee brings together local leaders in the technology, non-profit and education sectors to make sure our participants, champions, volunteers, and community have a positive experience and meaningful event.
```
Comment (csjoblom) [inline pages/meet-our-team/index.vue:12]: This likely needs to be broken up into multiple lines and will likely trigger a failed build on lint

**Comment by websupergirl** [comment]
Also was responsive tested for mobile and tablets.
Screen Shot 2019-11-18 at 5 41 10 PM
Screen Shot 2019-11-18 at 5 40 56 PM
Screen Shot 2019-11-18 at 5 40 43 PM

**Review by csjoblom — CHANGES_REQUESTED** [review]

File: pages/meet-our-team/index.vue (hunk 2 above)
Comment (websupergirl) [reply pages/meet-our-team/index.vue:12]: I ran the lintfix on it before pushing up ... ?

File: pages/meet-our-team/index.vue (hunk 1 above)
Comment (websupergirl) [reply pages/meet-our-team/index.vue]: I'll let her correct that in the issue when the time comes ... I'll need to push another update in about a week anyway as some of the people still haven't submitted their information.

**Comment by websupergirl** [comment]
I fixed the info for Frankie as she sent it while this was still open

File: pages/meet-our-team/index.vue (hunk 2 above)
Comment (csjoblom) [reply pages/meet-our-team/index.vue:12]: Ah ok that rule may have changed with the prettier removal, no worries then.

**Review by csjoblom — APPROVED** [review]

**Comment by csjoblom** [comment]
Push'd live

**Pull request merged**